debian/changelog

```diff
@@ -1,4 +1,4 @@
-livechart (1.9.1) noble; urgency=low
+livechart2 (1.9.1) noble; urgency=low
```

Comment:
Maybe we should add a draft changelog for livechart2 2.0.0 and keep `livechart` as it is here?

We kept the `granite` package name for the old GTK3 releases of Granite and use new `granite7` package name for GTK4 releases when we previously ported Granite into GTK4:

https://github.com/elementary/granite/blob/835f3b11b9eea0e8c94c0cd9e1a21bda45eae5a8/debian/changelog#L110